frontend/src/components/form-inputs/PasswordInput.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { FieldErrors } from 'react-hook-form';
+import FormFieldError from '../blocks/Error';
+import { EyeIcon } from "../../components/svg/Icons";
+
+interface InputProps {
+  label?: string;
+  required?: boolean;
+  errors: FieldErrors;
+  name: string;
+  type?: string;
```

Comment:
Probably don't need the 'type' prop since it will always be a password field in this case.